packages/device_info_plus/device_info_plus/example/integration_test/device_info_plus_test.dart

```diff
@@ -65,4 +66,121 @@ void main() {
+      expect(androidInfo.fingerprint, isNotNull);
+      expect(androidInfo.hardware, isNotNull);
+
+      expect(androidInfo.host, isNotNull);
+      expect(androidInfo.id, isNotNull);
+      expect(androidInfo.manufacturer, isNotNull);
+      expect(androidInfo.model, isNotNull);
+      expect(androidInfo.product, isNotNull);
+
+      expect(androidInfo.supported32BitAbis, isNotNull);
+      expect(androidInfo.supported64BitAbis, isNotNull);
+      expect(androidInfo.supportedAbis, isNotNull);
+
+      expect(androidInfo.tags, isNotNull);
+      expect(androidInfo.type, isNotNull);
+      expect(androidInfo.isPhysicalDevice, isNotNull);
+      expect(androidInfo.androidId, isNull);
+      expect(androidInfo.systemFeatures, isNotNull);
+    }
+  });
+
+  testWidgets('Windows', (WidgetTester tester) async {
+    if (Platform.isWindows) {
+      expect(windowsInfo.computerName, isNotNull);
+      expect(windowsInfo.numberOfCores, isNotNull);
+      expect(windowsInfo.systemMemoryInMegabytes, isNotNull);
+    }
+  });
+
+  testWidgets('WebBrowser', (WidgetTester tester) async {
+    if (kIsWeb) {
+      expect(webBrowserInfo.appCodeName, isNotNull);
+      expect(webBrowserInfo.appName, isNotNull);
+      expect(webBrowserInfo.appVersion, isNotNull);
+      expect(webBrowserInfo.deviceMemory, isNotNull);
+      expect(webBrowserInfo.language, isNotNull);
+      expect(webBrowserInfo.platform, isNotNull);
+      expect(webBrowserInfo.product, isNotNull);
+      expect(webBrowserInfo.productSub, isNotNull);
+      expect(webBrowserInfo.userAgent, isNotNull);
+      expect(webBrowserInfo.vendor, isNotNull);
+      expect(webBrowserInfo.vendorSub, isNotNull);
+      expect(webBrowserInfo.maxTouchPoints, isNotNull);
+      expect(webBrowserInfo.hardwareConcurrency, isNotNull);
+    }
+  });
+
+  testWidgets('Linux', (WidgetTester tester) async {
+    if (Platform.isLinux) {
+      expect(linuxInfo.name, isNotNull);
+      expect(linuxInfo.version, isNotNull);
+      expect(linuxInfo.id, isNotNull);
+      expect(linuxInfo.idLike, isNotNull);
+      expect(linuxInfo.versionCodename, isNotNull);
+      expect(linuxInfo.versionId, isNotNull);
+      expect(linuxInfo.prettyName, isNotNull);
+      expect(linuxInfo.buildId, isNotNull);
+      expect(linuxInfo.variant, isNotNull);
+      expect(linuxInfo.variantId, isNotNull);
+      expect(linuxInfo.machineId, isNotNull);
+    }
+  });
+
+  testWidgets('MacOS', (WidgetTester tester) async {
+    if (Platform.isMacOS) {
+      expect(macosInfo.computerName, isNotNull);
+      expect(macosInfo.hostName, isNotNull);
+      expect(macosInfo.arch, isNotNull);
+      expect(macosInfo.model, isNotNull);
+      expect(macosInfo.kernelVersion, isNotNull);
+      expect(macosInfo.osRelease, isNotNull);
+      expect(macosInfo.activeCPUs, isNotNull);
+      expect(macosInfo.memorySize, isNotNull);
+      expect(macosInfo.cpuFrequency, isNotNull);
+      expect(macosInfo.systemGUID, isNotNull);
+    }
+  });
```

Comment:
same as with the iOS, we don't have CI integration that runs tests on these platforms, so the test will never run. I'd suggest to remove them at the moment.